build.gradle

```diff
@@ -525,6 +525,7 @@ Import-Package: \
             include 'co/paralleluniverse/common/asm/**'
             include 'co/paralleluniverse/common/resource/**'
             include 'co/paralleluniverse/fibers/instrument/**'
+            exclude 'co/paralleluniverse/fibers/instrument/Classes*'
```

Comment:
Why are you excluding this new class from the instrumentor agent? Doesn't the agent need it? The `Task` classes are only excluded because they are Ant tasks.

Reply:
Because the test below this fails if I do not. I probably had not understood the subtlety of the dependencies here.

Reply:
It sounds like the project also needs a ProGuard task to ensure that the agent jar is self-consistent...